x-pack/plugins/ml/public/application/components/annotations/annotation_flyout/index.tsx

```diff
@@ -286,6 +318,23 @@ class AnnotationFlyoutUI extends Component<CommonProps & Props> {
                 value={annotation.annotation}
               />
             </EuiFormRow>
+            <EuiFormRow>
+              <EuiCheckbox
+                id={'xpack.ml.annotationFlyout.applyToPartition'}
+                label={
+                  <FormattedMessage
+                    id="xpack.ml.annotationFlyout.applyToPartitionTextLabel"
+                    defaultMessage="Apply annotation to this series"
+                  />
+                }
+                checked={this.state.applyAnnotationToSeries}
+                onChange={() =>
+                  this.setState({
+                    applyAnnotationToSeries: !this.state.applyAnnotationToSeries,
```

Comment:
When editing a series level annotation, this checkbox should be initialized with the checked state. For me, it is normally opening unchecked.

Reply:
This is now checked on by default here https://github.com/elastic/kibana/pull/70198/commits/1f2684dbbc91ca4a7d83ab93d8d4cfb79a5180c2

Reply:
As discussed, if editing an existing series level annotation, if the user then unticks the 'apply to series' checkbox, we should clear the partitioning and detector index fields of the annotation.

![image](https://user-images.githubusercontent.com/7405507/87057172-74950a80-c1fe-11ea-92da-7a7dd40c2929.png)

Reply:
@qn895 , I'm not sure if this screenshot is still current.  But I'm noticing the footer actions.  The 'Delete' action should be next to the 'Update' button instead of even spacing between all buttons.  You also do not need the cross next to 'Cancel'

For example... (checkmark not needed in your case)
![image](https://user-images.githubusercontent.com/3756330/87315628-9f2ede00-c4f2-11ea-9fac-54d4208157c0.png)

Reply:
Tested latest edits, and the partitioning and detector fields are now removed as expected if I untick the 'apply to series' checkbox when editing an annotation.